include/RevProc.h

```diff
@@ -696,6 +674,15 @@ class RevProc{
     return isFloat ? regFile->FP_Scoreboard.any() : regFile->RV_Scoreboard.any();
```

Comment:
As I said, if you need to test for both float and general registers, you can remove `isFloat` and change this function to return `regFile->RV_Scoreboard.any() || regFile->FP_Scoreboard.any()`.  (I put RV first because it's more likely in practice to be true, so short-circuiting of `||` will be faster.)